_posts/2021-06-10-external-domain-migration-announcement.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Migrate to the external-domain service"
+date: June 10, 2021
+excerpt: Customers need to take action to migrate from the cdn-service and custom-domain service to the external-domain service
+---
+
+## Who is this for?
+
+This post is for all users of the cdn-route or custom-domain services.
+
+## Background
+
+The custom-domain and cdn-route services leverage Let's Encrypt to provision certificates on our users' behalf.
+Several months ago, Let's Encrypt announced that they're deprecating their v1 API. To work with their new API, we've written a replacement
+to the cdn-route and custom-domain services, the external-domain service. We now need to migrate your service instances
+to the new service to ensure their certificates can continue renewing without issue. To accomplish this, we've written internal tooling to
+migrate instances automatically without service interruption, but it does require some action on your part to initiate the migration.
+
+## What you need to do
+
+The new external-domain service uses a different method of validation with Let's Encrypt. The new method relies on specific DNS records being created for your domain(s). To begin the migration, you need to create 2 DNS CNAME records for each domain on your custom-domain or cdn-route instances.
+
+| Type  | Name                      | Value                                                           |
+|-------|---------------------------|-----------------------------------------------------------------|
+| CNAME | `_acme-challenge.$DOMAIN` | `_acme-challenge.$DOMAIN.external-domains-production.cloud.gov` |
+| CNAME | `$DOMAIN`                 | `$DOMAIN.external-domains-production.cloud.gov`                 |
+
+The `_acme-challenge.$DOMAIN` CNAME record allows us to provision SSL certificates on your behalf.
+
+As an example, if you have a custom-domain service created for `directorate.agency.gov`, you'd want to create a CNAME record `_acme-challenge.directorate.agency.gov` with value `_acme-challenge.directorate.agency.gov.external-domains-production.cloud.gov`.
+
+The `$DOMAIN` CNAME is responsible for routing the user traffic to your site.  Using the `directorate.agency.gov` example above, this would be `directorate.agency.gov.external-domains-production.cloud.gov`.
+
```

Comment:
```suggestion

If you already have `CNAME`, `A`, and/or `AAAA` (with `ALIAS`) record(s) for `$DOMAIN`, you should update the value or replace the record to match what is shown above.

* If your domain is a "2nd level domain" (i.e. `agency.gov` instead of `directorate.agency.gov`) you will need to use an `A` and/or `AAAA` (with `ALIAS`) record(s) assuming your DNS provider supports it.

```

Reply:
I think this should be `and` rather than `and/or`